src/viewer.ts

```diff
@@ -282,7 +288,7 @@ class TrackableViewerState extends CompoundTrackable {
     this.add("statistics", viewer.statisticsDisplayState);
     this.add("helpPanel", viewer.helpPanelState);
     this.add("settingsPanel", viewer.settingsPanelState);
-    this.add("selection", viewer.selectionDetailsState);
+    this.add("selection", viewer.selectionDetailsState); // TODO, if I disable this, do we stop updating the state when we move the mouse?
```

Comment:
this should only trigger an update when the "pinned" state changes, but maybe there is a bug --- you could check